frontend/src/store/settings.ts

```diff
@@ -44,6 +51,18 @@ export default function useSettingsStore() {
     LocalStorage.set(settings.advancedMode, advancedMode.value);
   }
 
+  type Language = { label: string, value: string };
+
+  function setLanguage(newLanguage: Language) {
+    console.log(newLanguage)
+    language.value = newLanguage;
+
+    console.log('new set language value is', language.value.value)
```

Comment:
@mds1 Was expecting L60 to return 'cn-zh' instead of 'en-us'

Reply:
Looks like the issue is that you're missing a local `ref` to define the user's language selection. For example, in `NetworkDropdown.vue`, we use `currentNetwork` as the v-model param:

https://github.com/ScopeLift/umbra-protocol/blob/2a5c9dc47695a0cc57138bbad09243342d14c9fc/frontend/src/layouts/NetworkDropdown.vue#L3-L4

And `currentNetwork` is a ref that is returned from `setUp` (the ref should be defined within setup, instead of free-floating like it currently is, but not a big deal)

https://github.com/ScopeLift/umbra-protocol/blob/2a5c9dc47695a0cc57138bbad09243342d14c9fc/frontend/src/layouts/NetworkDropdown.vue#L26
https://github.com/ScopeLift/umbra-protocol/blob/2a5c9dc47695a0cc57138bbad09243342d14c9fc/frontend/src/layouts/NetworkDropdown.vue#L46

In `BaseLayout`, the `language` variable being used as the v-model parameter is the ref exported from settings. The default is set as en-us, and no matter what option the user does that default is then passed into `setNetwork` again